assignment4/sass/pages/_details.scss

```diff
@@ -0,0 +1,348 @@
+$report-bg-color : #ffffff;
+
+.middle-container {
+    width: 82%;
+    float: right;
+	.page-status {
+    	background-color: $report-bg-color;
+    	margin: 5px;
+    	padding-bottom:15px;
+    	padding-top:10px;
+		span {
+			margin-left:20px;
+		}
+	}
+	.report {
+		background-color: $report-bg-color;
+	    margin: 5px;
+	}
+	.test-report {
+		.header {
+    		padding-bottom:40px;
+			.header-text {
+				float: left;
+	    		width: 35%;
+	    		margin-left:20px;
+			}
+
+			h5 {
+				color:blue;
+			}
+
+			button {
```

Comment:
@bhartijain Don't add styling for `buttons` tag directly.  add basic button style through class and add the same classes where ever you need it.